src/test/java/htsjdk/variant/bcf2/BCF2WriterUnitTest.java

```diff
@@ -204,6 +198,50 @@ public void testWriteAndReadBCFHeaderless() throws IOException {
 
     }
 
+    /**
+     * test, using the writer and reader, that phased information is preserved in a round trip
+     */
+    @Test
+    public void testReadAndWritePhasedBCF() throws IOException {
+        final File vcfInputFile = new File("src/test/resources/htsjdk/variant/phased.vcf");
+        final File bcfOutputFile = File.createTempFile("testWriteAndReadBCFHeaderless.", ".bcf", tempDir);
+        final File vcfOutputFile = File.createTempFile("testWriteAndReadBCFHeaderless.", ".vcf", tempDir);
+        bcfOutputFile.deleteOnExit();
+        vcfOutputFile.deleteOnExit();
+
+        VCFFileReader vcfFile = new VCFFileReader(vcfInputFile);
+        VariantContextWriter bcfWriter = new VariantContextWriterBuilder().setOutputFile(bcfOutputFile).setReferenceDictionary(vcfFile.getFileHeader().getSequenceDictionary()).build();
```

Comment:
I would restructure this so that you create each of the writers in the try-with-resources block it's used in, and then have it be auto closed.